### PR TITLE
German import 

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- German import (Homepage, Essential Page, Use Ethereum, Use cases, Staking, Learn hub & guides, Roadmap, Community, Foundational docs, Foundational docs - nodes and clients)

### CODE REVIEW

File: public/content/translations/de/desci/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Dezentralisierte Wissenschaft (DeSci) ist eine Bewegung, die darauf abzielt, ein
...
 
-Dezentralisierte Wissenschaft ermöglicht eine Vielzahl von Finanzierungsmöglichkeiten (von [DAOs](/dao/), [quadratischen Spenden](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) bis hin zu Crowdfunding und mehr), einen leichteren Zugang zu Daten und Methoden sowie Anreize für Reproduzierbarkeit.
+Decentralized science allows for more diverse funding sources (from [DAOs](/dao/), [quadratic donations](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) to crowdfunding and more), more accessible data and methods, and by providing incentives for reproducibility.
```
Comment (minimalsm) [inline public/content/translations/de/desci/index.md:21]:
```suggestion
Dezentralisierte Wissenschaft ermöglicht eine Vielzahl von Finanzierungsmöglichkeiten (von [DAOs](/dao/), [quadratischen Spenden](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) bis hin zu Crowdfunding und mehr), einen leichteren Zugang zu Daten und Methoden sowie Anreize für Reproduzierbarkeit.
```

File: public/content/translations/de/developers/docs/networks/index.md [hunk 2]
```diff
@@ -56,12 +56,11 @@ Die beiden öffentlichen Testnets, die die Client-Entwickler derzeit betreiben,
 - [QuickNode Sepolia Faucet](https://faucet.quicknode.com/drip)
 - [Grabteeth](https://grabteeth.xyz/)
 - [PoW-Faucet](https://sepolia-faucet.pk910.de/)
-- [Sepolia-Faucet](https://faucet.sepolia.dev/)
-- [FaucETH](https://fauceth.komputing.org)
-- [Coinbase Wallet Faucet | Sepolia](https://coinbase.com/faucets/ethereum-sepolia-faucet)
-- [Alchemy Sepolia-Faucet](https://sepoliafaucet.com/)
-- [Infura Sepolia-Faucet](https://www.infura.io/faucet)
-- [Chainstack Sepolia-Faucet](https://faucet.chainstack.com/sepolia-faucet)
+- [Faucet für Coinbase-Wallet | Sepolia](https://coinbase.com/faucets/ethereum-sepolia-faucet)
```
Comment (minimalsm) [inline public/content/translations/de/developers/docs/networks/index.md:59]: The use of für here seems weird... 

Literally "Faucet for Coinbase-Wallet"

@lukassim any thoughts?

File: public/content/translations/de/roadmap/merge/index.md [hunk 3]
```diff
@@ -4,14 +4,13 @@ description: Erfahren Sie mehr über die Zusammenführung, als Mainnet Ethereum
...
 image: /upgrades/merge.png
-alt: 
 summaryPoint1: Ethereum Mainnet verwendet Proof-of-Stake, aber das war nicht immer der Fall.
 summaryPoint2: Der Wechsel vom ursprünglichen Proof-of-Work Mechanismus zu Proof-of-Stake wurde Zusammenführung genannt.
 summaryPoint3: Die Zusammenführung bezieht sich auf das ursprüngliche Ethereum Mainnet, welches mit einer separaten Proof-of-Stake-Blockchain namens Beacon Chain vereinigt wurde, und somit nun beide als eine Blockchain existieren.
 summaryPoint4: Die Zusammenführung reduzierte Ethereums Energieverbrauch um ca. 99,95 %.
 ---
 
-<UpgradeStatus dateKey="page-upgrades:page-upgrades-beacon-date">
+<UpgradeStatus dateKey="page-upgrades-beacon-date">
```
Comment (minimalsm) [inline public/content/translations/de/roadmap/merge/index.md:13]:
```suggestion
<UpgradeStatus dateKey="page-upgrades:page-upgrades-beacon-date">
```

File: src/intl/de/page-gas.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
+{
+  "page-gas-meta-title": "Gas fees on Ethereum: how do they work?",
```
Comment (minimalsm) [inline src/intl/de/page-gas.json:2]: Similar to @wackerow's comments elsewhere, we should remove this! :)

File: public/content/translations/de/roadmap/beacon-chain/index.md [hunk 5]
```diff
@@ -4,13 +4,12 @@ description: Informieren Sie sich über die Beacon Chain – das Upgrade, mit de
...
 template: upgrade
 image: /upgrades/core.png
-alt: 
 summaryPoint1: Mit der Bacon Chain wurde Proof-of-Stake in das Ethereum Ökosystem eingeführt.
 summaryPoint2: Sie wurde 2022 mit der ursprünglichen Ethereum Proof-of-Work Blockchain vereinigt.
 summaryPoint3: Die Beacon Chain führte die Konsenslogik und das Block-Gossip-Protokoll ein, das nun Ethereum sichert.
 ---
 
-<UpgradeStatus isShipped dateKey="page-upgrades:page-upgrades-beacon-date">
+<UpgradeStatus isShipped dateKey="page-upgrades-beacon-date">
```
Comment (minimalsm) [inline public/content/translations/de/roadmap/beacon-chain/index.md:12]:
```suggestion
<UpgradeStatus isShipped dateKey="page-upgrades:page-upgrades-beacon-date">
```

**Review by minimalsm — APPROVED** [review]
LGTM other than the changes I tagged @lukassim [on here](https://github.com/ethereum/ethereum-org-website/pull/11976#discussion_r1466556229). 

Not blocking this PR necessarily.

**Review by wackerow — APPROVED** [review]

**Pull request merged**